lib/load_csv_helper.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
```

Comment:
Do you think we still need the load_csv_helper. I am not sure that this rake task would be terribly useful anymore, perhaps we should ignore it?